app/src/main/java/org/mozilla/focus/shortcut/IconGenerator.kt

```diff
@@ -76,21 +77,22 @@ class IconGenerator {
 
         private fun drawCharacterOnBitmap(context: Context, character: Char, bitmap: Bitmap): Bitmap {
             val canvas = Canvas(bitmap)
-
             val paint = Paint()
+            val rect = Rect()
 
             val textSize = TypedValue.applyDimension(
                     TypedValue.COMPLEX_UNIT_DIP, TEXT_SIZE_DP, context.resources.displayMetrics)
 
             paint.color = Color.WHITE
-            paint.textAlign = Paint.Align.CENTER
+            paint.textAlign = Paint.Align.LEFT
             paint.textSize = textSize
             paint.isAntiAlias = true
 
-            canvas.drawText(character.toString(),
-                    canvas.width / 2.0f,
-                    canvas.height / 2.0f - (paint.descent() + paint.ascent()) / 2.0f,
-                    paint)
+            paint.getTextBounds(character.toString(), 0, 1, rect)
```

Comment:
nit: micro-optimization, but maybe we should cache `character.toString()` in a local variable?